assets/css/style.css

```diff
@@ -0,0 +1,566 @@
+  line-height: 44px;
+  color: #fff;
+  cursor: pointer;
+  padding-left: 0.5em;
+  margin-bottom: 1em;
+  text-decoration: none;
+}
+
+.nav-items.show a:first-child {
+  text-align: end;
+  padding-right: 1em;
+}
+
+.menubar {
+  background-color: transparent;
+  margin-right: 0.5em;
+}
+
+.menubar > img {
+  max-height: 2em;
+}
+
+main {
+  min-height: 85vh;
+}
+
+#welcome {
+  padding: 5em 2em;
+  background-image: url('../images/backgrounds/about_bg_01.png');
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+#welcome h1 {
+  font-size: 3rem;
+  font-weight: 800;
+  line-height: 3rem;
+
+  /* background-clip: initial; */
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-image: -webkit-linear-gradient(45deg, #f40001, #ec5242, rgb(255, 64, 0), rgb(255, 123, 0), #a00000, #f65112, #ec5242);
+}
+
+.card {
+  border-radius: 1em;
+  background-color: #fff;
+  padding: 1em;
+  margin: 1em;
+  margin-bottom: 5em;
+}
+
+.grid-row {
+  display: grid;
+  grid-template:
+    '.'
+    '.'
+    '.';
+}
+
+.speaker-block {
+  display: grid;
+  grid-template: 'itemImage itemDesc itemDesc itemDesc';
+  gap: 1em;
+  grid-auto-columns: minmax(0, 1fr);
+}
+
+.speaker-block:hover {
+  border: #000 1px solid;
+  padding: 1em;
+  border-radius: 1em;
+}
+
+.grid-col {
+  display: grid;
+  grid-template: 'itemImage itemDesc';
+  gap: 1em;
+  grid-auto-columns: minmax(0, 1fr);
+}
+
+.speaker-image > img {
+  max-width: 100%;
+}
+
+.grid-col-item-image,
+.speaker-image {
+  grid-area: itemImage;
+  max-height: 8em;
+}
+
+.grid-col-item-desc,
+.speaker-desc {
+  grid-area: itemDesc;
+}
+
+.speaker-desc {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5em;
+}
+
+.see-more-speakers {
+  max-width: 100%;
+  margin: 2em 1em;
+}
+
+.see-more-speakers > button {
+  box-shadow: 2px 2px 4px 0 #00000040;
+  border: 1px solid #dcdcdc;
+  background-color: #fff;
+  color: #000;
+  border-radius: 4px;
+  width: 100%;
+  padding: 1em;
+  cursor: pointer;
+}
+
+.flex-list {
+  display: inline-flex;
+  padding: 0;
+  margin-top: 0.3em;
+}
+
+.hidden,
+.desktop {
+  display: none;
+}
+
+.mobile {
+  display: block;
+}
+
+.w-100 {
+  width: 100%;
+}
+
+.warning {
+  color: #ac0b0b;
+  font-size: larger;
+}
+
+.brief {
+  font-family: Lato, serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 17px;
+  letter-spacing: -0.01em;
+  text-align: left;
+  color: #515151;
+  border: solid #fff 2px;
+  padding: 1em;
+  margin: 2em 0;
+}
+
+.event-time {
+  font-family: Lato, serif;
+  font-size: x-large;
+  font-weight: bolder;
+  line-height: 36px;
+  letter-spacing: -0.01em;
+  text-align: left;
+  color: #000;
+
+  /* padding: 1em; */
+  margin: 0.5em 0;
+}
+
+.event-venue {
+  font-family: Lato, serif;
+  font-size: large;
+  font-weight: 400;
+  line-height: 36px;
+  letter-spacing: -0.01em;
+  text-align: left;
+  color: #515151;
+}
+
+#program {
+  background: #282b32;
+  color: #fff;
+  text-align: center;
+}
```

Comment:
- [ ] Your design is really looking great. Kudos! However, I can see you have used a lot of colors throughout you stylesheet to match the design which is quite impressive and innovative. But you are suppose to use the exact same colors, for text, backgrounds, borders, etc, from the original design. 

The main colors from the original design are: `#272A31`, `#EC5242`, `#D3D3D3`.
You can use just these three colors with varying degrees of opacity to make your project more beautiful and match the actual design 

- [ ] You seem to have about 5 stylesheets. I would advise you to consider using one stylesheet or at most 2 stylesheet (one per page) since it is a mobile first approach. This approach can offer several benefits, including improved modularity, easier maintenance, and reduced risk of style conflicts. It can also help you conform to the standard best practices.